usr/lib/ica_s390_stdll/ica_specific.c

```diff
@@ -484,6 +510,9 @@ static CK_BBOOL ica_need_wr_lock_rsa_privkey(OBJECT *obj, void *ex_data,
     if (ex_data == NULL || ex_data_len < sizeof(ica_private_data_t))
         return FALSE;
 
+    if (data->blinding == NULL || data->blinding_mont_ctx == NULL)
+        return TRUE;
```

Comment:
Do we need to return TRUE? Or is it FALSE like above?

Reply:
We need to return TRUE, because when blinding or blinding_mont_ctx  is NULL, then we need to setup blinding first, and this requires a write lock, since we are changing the attached data (i.e. set the blinding and blinding_mont_ctx fields). 

Similar when modexpoKey and crtKey are both NULL, then we also need a write lock because we are updating one of them.